src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx

```diff
@@ -586,4 +614,123 @@ const Part: FunctionComponent<{
     </div>
   )
 }
+
+const Articles = ({articles, children}) => {
```

Comment:
```suggestion
const Articles: React.FC<{articles: any}> = ({articles, children}) => {
```

You can't use `any` implicitly and have to be explicit about it like this. Even better to actually type it.

Reply:
note that `children` is typed in `React.FunctionComponent` so no need to double up on it.

Reply:
This makes sense. I'll make sure to be explicit about it.

Reply:
more to the point TypeScript will make sure youre explicit about it lol 😅